keyboards/annepro2/keymaps/default/keymap.c

```diff
@@ -80,7 +80,7 @@ enum anne_pro_layers {
   *
   */
  [_FN2_LAYER] = KEYMAP( /* Base */
-    KC_TRNS, KC_AP2_BT1, KC_AP2_BT2, KC_AP2_BT3, KC_AP2_BT4, KC_TRNS, KC_TRNS, KC_TRNS, KC_AP_LED_OFF, KC_AP_LED_ON, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+    KC_TRNS, KC_AP2_BT1, KC_AP2_BT2, KC_AP2_BT3, KC_AP2_BT4, KC_TRNS, KC_TRNS, KC_TRNS, KC_AP_LED_OFF, KC_AP_LED_ON, KC_AP_LED_NEXT_INTENSITY, KC_TRNS, KC_TRNS, KC_TRNS,
```

Comment:
added a shortcut `fn2+F10` to test the intensity functionality in the default keymap.